src/Microsoft.AspNet.Mvc.Razor/IRazorPage.cs

```diff
@@ -20,9 +19,9 @@ public interface IRazorPage
         ViewContext ViewContext { get; set; }
 
         /// <summary>
-        /// Gets or sets the action invoked to render the body.
+        /// Gets or sets the body content.
         /// </summary>
-        Func<TextWriter, Task> RenderBodyDelegateAsync { get; set; }
+        IHtmlContent BodyContent { get; set; }
```

Comment:
This property is the buffered content that gets passed to a layout page, not super sure what it would look like if we were to turn it into a method.